flags/pom.xml

```diff
@@ -59,7 +59,18 @@
             <classifier>no_aop</classifier>
             <scope>provided</scope>
         </dependency>
-
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>container-dev</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>zkfacade</artifactId>
```

Comment:
would be great to avoid new dependencies in flags

Reply:
I tried to move `FlagsDb` back to _configserver-flags_, but `FlagsHandlerTest` depends on `FlagsDbImpl` and `MockCurator`. It's provided though, so it doesn't affect the size of the _flags_ bundle.

I think this issue may be better solved by introducing a lightweight _flags-api_ containing `FlagSource` etc.

Reply:
Not sure I understand the issue. FlagsHandlerTest is in configserver which already depends on configserver-flags, so moving flagsDb(Impl) to configserver-flags should be OK?

Reply:
The REST API and associated tests have been moved to _flags_ so that it can be reused by _controller-server_. That's the whole point of this PR 🙂